style/js/shield_defs.js

```diff
@@ -384,6 +384,48 @@ export function loadShields(shieldImages) {
     },
   };
 
+  ["COL", "JEF", "MAH", "OTT", "SEN", "STA", "SUM", "TUS"].forEach(
+    // Yellow on blue pentagon
+    (county) => (shields["US:OH:" + county] = usMUTCDCountyShield)
+  );
+  ["ATH", "BEL", "GUE", "HAR", "HEN", "MOE", "PAU", "WAS", "WIL", "WYA"].forEach(
+    // White on green rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#006747", "white", "white", 2, 1, null))
+  );
+  ["MED", "NOB"].forEach(
+    // White on blue rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#003f87", "white", "white", 2, 1, null))
+  );
+  ["TRU", "VIN"].forEach(
+    // Black on yellow rectangle
+    (county) => (shields["US:OH:" + county] = roundedRectShield("#ffcd00", "black", "black", 2, 1, null))
+  );
+  shields["US:OH:ASD"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_oh_asd,
+    ],
+    textColor: "#006747",
+    padding: {
+      left: 6,
+      right: 3,
+      top: 4,
+      bottom: 7,
+    },
+  };
+  shields["US:OH:SCI"] = {
+    backgroundImage: [
+      shieldImages.shield40_us_oh_sci_2,
+      shieldImages.shield40_us_oh_sci_3,
+    ],
+    textColor: "black",
+    padding: {
+      left: 3,
+      right: 3,
+      top: 4,
+      bottom: 6,
+    },
+  };
```

Comment:
I put an orange stroke around the Scioto County shape, even though the shield gives it a black stroke on an orange field. A black stroke would give it much better contrast against the background but less contrast against nearby Ohio state route shields.

[<img src="https://upload.wikimedia.org/wikipedia/commons/1/18/Scioto_County_Route_40_OH.svg" width="64" alt="Scioto CR 40">](https://commons.wikimedia.org/wiki/File:Scioto_County_Route_40_OH.svg)